templates/docs/patterns/images.md

```diff
@@ -4,10 +4,46 @@ context:
   title: Images | Components
 ---
 
-Enhance images by adding a variant style with a border or drop shadow.
+Enhance images by distinguishing them from their background or controlling their container's aspect ratio.
+
+## Highlighted image
+
+Insulate an image from its surroundings. This can help to emphasize the image
+and/or prevent its white space from becoming indistinguishable from the surrounding white space.
+
+<div class="embedded-example"><a href="/docs/examples/patterns/image/container/highlighted" class="js-example">
+View example of the image container with a highlighted background
+</a></div>
+
+## Image container with aspect ratio
+
+You can modify the aspect ratio of an image container to neatly wrap an image in a container of the desired aspect ratio.
+This is useful for aligning images or other content with mismatching aspect ratios.
+
+All image containers center the `.p-image-container__image` element inside them by default.
+If you need to change image alignment within the image container, use the [image position utility](/docs/utilities/image-position).
+
+| CSS Class                      | Aspect ratio |
```

Comment:
That reminded me that we actually have a feature to generate list of component class names from the code comment, like this: https://vanillaframework.io/docs/base/code#class-reference

Maybe it would be relatively simple to add something like that to image container? It's basically a yaml at the beginning of the scss file:
https://github.com/canonical/vanilla-framework/blob/main/scss/_patterns_code-snippet.scss#L1-L45

Reply:
Revised to use class reference for classname documentation